.prettierignore

```diff
@@ -0,0 +1 @@
+**/*/CHANGELOG.md
```

Comment:
Disable prettier.format() for CHANGELOG.md in ModuleGenerator https://github.com/aws/aws-sdk-js-v3/blob/379a5c11f4620d974ba201c16c0453e0cc82fe29/packages/package-generator/src/ModuleGenerator.ts#L57

Reply:
Nice catch! Will update it.